src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java

```diff
@@ -51,8 +51,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             }
         }
 
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        if (!player.isOp())
+        {
+            TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+            player.setOp(true);
+        }
+        else
+        {
+            playerMsg("That player is already OP.");

```

Comment:
This should read "That player is already an OP." - we might as well use correct English ;)